src/components/Exercise/List.js

```diff
@@ -25,15 +35,26 @@ export default function List({ exercises, handleExerciseSelect }) {
     }
     return acc
   }, [])
-
   return (
     <>
       <S.ButtonWrapper>
-        <Button as={Link} to={`${url}/add`}>
+        <Button primary="true" as={Link} to={`${url}/add`}>
           Add
         </Button>
+        <SearchBox
+          register={register}
+          onSubmit={handleSubmit(onSubmit)}
+          reset={reset}
+          search={search}
+        />
       </S.ButtonWrapper>
-      <S.Wrapper>{renderExercises}</S.Wrapper>
+      <S.Wrapper>
+        {exercises.length !== 0 ? (
+          renderExercises
```

Comment:
Maybe rename to `renderedExercises`? As it is, it sounds like a function name. Also beware, you check the length of `exercises` and then render `renderExercises`, seems like a bug..

Reply:
you are right, renderedExercises makes more sense, changed it 
The check for exercises is tied to my search bar, because it changes my exercises array  I can show an error when no exercise is found